utils/clflags.ml

```diff
@@ -125,6 +125,7 @@ and opaque = ref false                  (* -opaque *)
 
 and dump_cmm = ref false                (* -dcmm *)
 let dump_selection = ref false          (* -dsel *)
+let dump_combine = ref false            (* -dcombine *)
```

Comment:
If you're moving things to better fit the order of the passes, you might want to move `dump_interval` somewhere around `-dregalloc` (between `-dprefer` and `-dregalloc` would seem the most appropriate to me)

Reply:
Done, thanks!